src/ztex/ztex.c

```diff
@@ -468,6 +496,12 @@ int ztex_scan_new_devices(struct ztex_dev_list *new_dev_list, struct ztex_dev_li
 	}
 
 	libusb_free_device_list(usb_devs, 1);
+
+	if (warn_open && num_fail_open) {
+		fprintf(stderr, "Warning: unable to access %d board(s), that could be"
+			" insuffisient permissions\nor other john invocation is running.\n",
```

Comment:
Should be spelled "insufficient". Also let's close and re-open the quotes after "\n" for readability. Something like this:

```c
		fprintf(stderr, "Warning: unable to access %d board(s), could be "
			"insufficient permissions or\n"
			"another instance of john running\n",
```